internal_testsuites/golang/test_helpers/test_helpers.go

```diff
@@ -114,6 +114,10 @@ var (
 	emptyEntrypointArgs          = []string{}
```

Comment:
lets attack the typescript tests too

Reply:
https://github.com/kurtosis-tech/kurtosis/pull/1731/files#diff-c9cd743933a35ab465e30df7562f3a08745e81c93917547a3fbdf3d1e89ce84c test.skip should work